distribution/src/database/openfire_db2.sql

```diff
@@ -192,34 +192,35 @@ CREATE TABLE ofMucServiceProp (
 );
 
 CREATE TABLE ofMucRoom (
-  serviceID           INTEGER       NOT NULL,
-  roomID              INTEGER       NOT NULL,
-  creationDate        CHAR(15)      NOT NULL,
-  modificationDate    CHAR(15)      NOT NULL,
-  name                VARCHAR(50)   NOT NULL,
-  naturalName         VARCHAR(255)  NOT NULL,
-  description         VARCHAR(255),
-  lockedDate          CHAR(15)      NOT NULL,
-  emptyDate           CHAR(15),
-  canChangeSubject    INTEGER       NOT NULL,
-  maxUsers            INTEGER       NOT NULL,
-  publicRoom          INTEGER       NOT NULL,
-  moderated           INTEGER       NOT NULL,
-  membersOnly         INTEGER       NOT NULL,
-  canInvite           INTEGER       NOT NULL,
-  roomPassword        VARCHAR(50),
-  canDiscoverJID      INTEGER       NOT NULL,
-  logEnabled          INTEGER       NOT NULL,
-  subject             VARCHAR(100),
-  rolesToBroadcast    INTEGER       NOT NULL,
-  useReservedNick     INTEGER       NOT NULL,
-  canChangeNick       INTEGER       NOT NULL,
-  canRegister         INTEGER       NOT NULL,
-  allowpm             INTEGER       NULL,
-  fmucEnabled         INTEGER       NULL,
-  fmucOutboundNode    VARCHAR(255)  NULL,
-  fmucOutboundMode    INTEGER       NULL,
-  fmucInboundNodes    VARCHAR(2000) NULL,
+  serviceID                     INTEGER       NOT NULL,
```

Comment:
The scripts use indentation for alignment. Since preserveHistOnRoomDeletion is a longer name, the indentation was adjusted. Don't you want to preserve alignment?

Reply:
Maybe use a shorter name?

Reply:
Ok, I've added a shorter name.